scripts/start.sh

```diff
@@ -1,9 +1,21 @@
 #!/bin/bash -e
 
-export ANDROID_API_FOR_ABI_32=24
-export ANDROID_API_FOR_ABI_64=24
+export ANDROID_API_FOR_ABI_32=29
+export ANDROID_API_FOR_ABI_64=29
```

Comment:
```suggestion
export ANDROID_API_FOR_ABI_32=24
export ANDROID_API_FOR_ABI_64=24
```

would be good to keep the minSdkVersion aligned with react-native

Reply:
in the BabylonNative embedding, I need it to meet the requirements of shipping on Android XR devices. We can give this a try and see if it works. if it doesn't, maybe we can produce a separate build artifact.